test/libsolidity/syntaxTests/errors/require_custom_legacy.sol

```diff
@@ -0,0 +1,8 @@
+error E(uint a, uint b);
+contract C {
+    function f(bool c) public pure {
+        require(c, E(2, 7));
+    }
+}
+// ----
+// UnimplementedFeatureError: Require with a custom error is only available using the via-ir pipeline.
```

Comment:
Curious - we don't need ``compileViaYul: false`` here?

Reply:
No, it's `false` by default.

Reply:
By the way, is this along the lines of how you suggested to implement the disallowing of the feature in legacy? The test suite handles the exception by printing the above, however, compiling it via CLI wil just spew out the exception, which is not great.